documentation/user_docs/docs/manual/Cutting_data_of_interest_from_SQW_files_and_objects.rst

```diff
@@ -159,47 +163,34 @@ Each can independently have one of four different forms below.
    to ensure your cut is what you expect.
 
 
-* ``[]``
-
-  An empty binning range will use the source binning axes in that dimension.
```

Comment:
Incorrect. The same algorithm works regardless of projection chaned. Correct description recovered